helm/hpcc/values.yaml

```diff
@@ -101,9 +101,8 @@ security:
 
 ## storage:
 ##
-## 1. If a component has the storagePlane property set, then that plane will be the default data location for that component.
-## 2. If storage.dataStorage.plane is defined then that plane will be the default location
-## 3. If there is a plane definition with no label or a label of "data" then the first matching plane will be the default data location
+## 1. If a component has the dataPlane property set, then that plane will be the default data location for that component.
```

Comment:
should this be clarified to say that this is only relevant for the engine components?
otherwise people might think it's worth [trying] to set to others where it is not relevant.